configs/sst_idm_sssd-python-pam.yaml

```
@@ -0,0 +1,11 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: python-pam
+  description: Python bindings for libpam.
+  maintainer: sst_idm_sssd
+  packages:
+    - python-pam
```

This needs to be the name of the binary RPM, not the SRPM, iow:
```suggestion
    - python3-pam
```

---

Updated